projects/plugins/jetpack/_inc/jetpack-modules.models.js

```diff
@@ -50,15 +48,18 @@ window.jetpackModules.models = ( function ( window, $, _, Backbone ) {
 			}
 
 			if ( m_sort.data( 'sort-by' ) ) {
-				items = _.sortBy( items, m_sort.data( 'sort-by' ) );
-				if ( 'reverse' === m_sort.data( 'sort-order' ) ) {
-					items.reverse();
-				}
+				const key = m_sort.data( 'sort-by' );
+				const cmpret = 'reverse' === m_sort.data( 'sort-order' ) ? -1 : 1;
+
+				items.sort( ( a, b ) =>
+					// eslint-disable-next-line no-nested-ternary
+					a[ key ] > b[ key ] ? cmpret : a[ key ] < b[ key ] ? -cmpret : 0
+				);
 			}
 
 			// Sort unavailable modules to the end if the user is running in local mode.
-			groups = _.groupBy( items, 'available' );
-			if ( _.has( groups, 'false' ) ) {
+			groups = Object.groupBy( items, item => item.available );
```

Comment:
TIL about `Object.groupBy()`. It appears [relatively new](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/groupBy); how far back do we need to support?

I wonder how it'd handle situations where there are no modules available (concat with undefined seems interesting).

That said, it seems this is just sorting everything, so perhaps a simple sort would do?

Reply:
> how far back do we need to support?

Our browserslist config currently gives
```
and_chr 138
android 138
chrome 138
chrome 137
chrome 136
chrome 112
edge 138
edge 137
firefox 140
firefox 139
ios_saf 18.5
ios_saf 18.4
opera 117
opera 116
safari 18.5
safari 18.4
samsung 28
```
Surprisingly old version of Chrome in there. Sigh.

> I wonder how it'd handle situations where there are no modules available (concat with undefined seems interesting).

🤷 Existing bug.

Reply:
Good call on just using `sort`, it's stable since 2019 (and in all those browsers) so we should be good.

Reply:
But the code above isn't relying on the sort being [stable](https://en.wikipedia.org/wiki/Sorting_algorithm#Stability), while this very much is.